src/main/java/org/broadinstitute/consent/http/ConsentApplication.java

```diff
@@ -368,8 +368,17 @@ private void initializeLiquibase(ConsentConfiguration config) throws LiquibaseEx
                 config.getDataSourceFactory().getPassword()
         );
         Database database = DatabaseFactory.getInstance().findCorrectDatabaseImplementation(new JdbcConnection(connection));
-        Liquibase liquibase = new Liquibase("changelog-master.xml", new ClassLoaderResourceAccessor(), database);
+        Liquibase liquibase = new Liquibase(liquibaseFile(), new ClassLoaderResourceAccessor(), database);
         liquibase.update(new Contexts(), new LabelExpression());
     }
 
+    private String liquibaseFile() {
+        String changeLogFile = System.getenv("CONSENT_CHANGELOG_FILE");
+        if (changeLogFile == null || changeLogFile.trim().isEmpty()) {
```

Comment:
minor nit, I prefer `Objects.nonNull()` for null checks
```suggestion
        if (Objects.nonNull(changeLogFile) || changeLogFile.trim().isEmpty()) {
```